electron/main.ts

```diff
@@ -955,14 +998,15 @@ ipcMain.handle('importGame', async (event, args) => {
       notify({ title, body: i18next.t('notify.install.canceled') })
       mainWindow.webContents.send('setGameStatus', {
         appName,
+        runner,
         status: 'done'
       })
       logInfo(err, LogPrefix.Backend)
     })
 })
 
-ipcMain.handle('updateGame', async (e, game) => {
-  if (!(await isOnline())) {
+ipcMain.handle('updateGame', async (e, game, runner) => {
+  if (!(await isOnline()) && runner == 'legendary') {
```

Comment:
why don't we have the same behavior for GOG as well since we will not be able to update a game being offline?
I noticed in other parts as well that need to be only that we are skipping the check.
would be good to test Heroic without internet connection to see how it will behave.

Reply:
This is my mistake actually. I'll remove that runner check